src/Manager/CollectionManager.php

```diff
@@ -79,7 +79,8 @@ public function createCollection($collectionDefinitionName)
             $definition['default_sorting_field'],
             $tokenSeparators,
             $symbolsToIndex,
-            $definition['enable_nested_fields'] ?? false
+            $definition['enable_nested_fields'] ?? false,
+            $definition['embed'] ?? null
```

Comment:
related to my previous comment, there is no embed option available at the collection definition level.
```
Unrecognized option "embed" under "acseo_typesense.collections.rfps". Available options are "collection_name", "default_sorting_field", "enable_n
  ested_fields", "entity", "fields", "finders", "symbols_to_index", "token_separators".
```